src/server/general.v

```
@@ -106,6 +106,12 @@ pub fn (mut ls LanguageServer) initialized(mut wr ResponseWriter) {
 
 	if need_index_stdlib {
 		ls.indexing_mng.indexer.add_indexing_root(ls.paths.vmodules_root, .modules, ls.paths.cache_dir)
+		for path in os.vmodules_paths()[1..] {
```

```suggestion
		for path in os.vmodules_paths()#[1..] {
```

---

Just `[1..]` will panic, when there is only one path.

---

For now, an empty array is returned when the slice start index equals the length of the array. The `os.vmodules_paths()` 
 api always return an array of length greater than or equal to 1.So it's ok.But in other words, array slicing should also report an error when generating a new array which the index is actually equal to the length of the array. I don't know if this idea has been followed up.or any incorrect slice returns the empty array instead of using a `#` symbol.Just ideas.😊

---

> The os.vmodules_paths() api always return an array of length greater than or equal to 1. So it's ok.

Yes.